src/main/java/core/basesyntax/Main.java

```diff
@@ -7,11 +7,8 @@ public static void main(String[] args) {
         Figure[] figurs = new Figure[randomLength];
         for (int i = 0; i < figurs.length; i++) {
             figurs[i] = figureSupplier.getRandomFigure();
-        }
-        for (Figure figur : figurs) {
-            if (figur != null) {
-                System.out.println(figur.getDrow());
-            }
+            System.out.println(figurs[i].drow());
```

Comment:
```suggestion
            figurs[i].drow();
```
`drow()` should have `System.out.println` in it